### PR TITLE
Add Kaiju Bunny

### PR DESCRIPTION
**Pull request by tmagicturtle**

### CODE REVIEW

**Comment by tmagicturtle** [comment]
Forgot to add the ability; do not merge

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -112,6 +112,47 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Normal",
 	},
 
+	// Kaiju Bunny
+	cozycuddle: {
+		accuracy: 95,
+		basePower: 0,
+		category: "Status",
+		desc: "Traps the target and lowers its Attack and Defense by two stages.",
+		shortDesc: "Traps target. Atk -2 Def -2.",
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
		shortDesc: "Target: trapped, Atk and Def lowered by 2.",
```

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -112,6 +112,47 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Flatter', target);
+			this.add('-anim', source, 'Let\'s Snuggle Forever', target);
+		},
+		onHit(target, source, move) {
+			this.boost({atk: -2, def: -2}, target, target);
+		},
+		effect: {
+			onStart(pokemon, source) {
+				this.add('-start', pokemon, 'move: Cozy Cuddle', '[of]' + source);
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]: shouldn't this be `source.name`

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -112,6 +112,47 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+			},
+			onTrapPokemon(pokemon) {
+				if (this.effectData.source && this.effectData.source.isActive) pokemon.tryTrap();
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
				if (this.effectData.source?.isActive) pokemon.tryTrap();
```

File: data/mods/ssb/random-teams.ts [hunk 4]
```diff
@@ -46,6 +46,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Data Corruption',
 		evs: {hp: 4, spa: 252, spe: 252}, nature: 'Timid',
 	},
+	'Kaiju Bunny': {
+		species: 'Lopunny', ability: 'Second Wind', item: 'Lopunnite', gender: 'F',
+		moves: ['Return', 'Play Rough', ['Drain Punch', 'High Jump Kick']],
+		signatureMove: 'Cozy Cuddle',
+		evs: {atk: 252, spd: 4, spe: 252}, nature: 'Jolly', level: 100, shiny: 1,
```
Comment (KrisXV) [inline data/mods/ssb/random-teams.ts]:
```suggestion
		evs: {atk: 252, spd: 4, spe: 252}, nature: 'Jolly', shiny: 1,
```

File: data/mods/ssb/statuses.ts [hunk 5]
```diff
@@ -69,6 +69,18 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 			this.add(`c|${getName('GXS')}|A Critical Error Has Occurred. Would You Like To Send A Report? Sending Report.`);
 		},
 	},
+	kaijubunny: {
+		noCopy: true,
+		onStart() {
+			this.add(`c|${getName('Kaiju Bunny')}|I heard SOMEONE wasn't getting enough affection! ￣( ÒㅅÓ)￣`);
```
Comment (KrisXV) [inline data/mods/ssb/statuses.ts:75]:
```suggestion
			this.add(`c|${getName('Kaiju Bunny')}|I heard SOMEONE wasn't getting enough affection! ￣( ÒㅅÓ)￣`);
			if (source.species.id !== 'lopunnymega' || source.illusion) return;
			this.add('-start', source, 'typechange', 'Normal/Fairy');
```

**Review by KrisXV — CHANGES_REQUESTED** [review]
In `scripts.ts`, ctrl + f Overneat and then add this code below overneat's mega code:

```js
if (pokemon.name === 'Kaiju Bunny' && !pokemon.illusion) this.add('-start', pokemon, 'typechange', 'Normal/Fairy');
```

File: data/mods/ssb/moves.ts (hunk 2 above)
Comment (tmagicturtle) [reply data/mods/ssb/moves.ts]: I'll change it, but that's how it was in Octolock

**Pull request merged**